pkg/kube/inject/webhook.go

```diff
@@ -755,6 +760,8 @@ func postProcessPod(pod *corev1.Pod, injectedPod corev1.Pod, req InjectionParame
 		return err
 	}
 
+	replaceProxyRunAsUserID(pod, req.proxyUID)
```

Comment:
I don't think we need to call this anymore, the func can be removed IIUC (see below)